testnet/testnet_test.go

```diff
@@ -56,30 +56,27 @@ func buildAndRunTestRunner(t *testing.T,
 ) {
 	t.Helper()
 
+	// bring up the test runner
 	t.Log("building and deploying test runner")
+	testRunner, err := pool.RunWithOptions(
+		&dockertest.RunOptions{
+			Name:       "test_runner",
+			Repository: "test-runner",
+			Tag:        "prebuilt",
+			NetworkID:  network.Network.ID,
+			PortBindings: map[docker.Port][]docker.PortBinding{
+				"8545/tcp": {{HostIP: "", HostPort: "8545"}},
+			},
+			Mounts: []string{
+				fmt.Sprintf("%s/testdata:/testdata", wd),
+			},
+			Env: []string{
+				"RUST_BACKTRACE=full",
+				"RUST_LOG=INFO",
+				fmt.Sprintf("TEST_TYPE=%s", testType),
```

Comment:
Where does `testType` resolve to what the rust test_runner is looking for? 

I believe it only understands these exact values:
- "ORCHESTRATOR_KEYS"
- "ARBITRARY_LOGIC"
- "VALIDATOR_OUT"
- "BATCH_STRESS"
- "VALSET_STRESS"
- "HAPPY_PATH",
- "V2_HAPPY_PATH"

Is there some place the camelized matrix names are translated?

Reply:
Here's the code that consumes TEST_TYPE, for reference:
https://github.com/PeggyJV/gravity-bridge/blob/mvid/converge-tests/orchestrator/test_runner/src/main.rs#L158

Reply:
I see how this is resolved now; resolving my convo. 🚢